pkg/controller/certificates/certificate_controller.go

```diff
@@ -209,12 +211,14 @@ func (cc *CertificateController) syncFunc(key string) error {
 // it's spammy and usually user error. Instead we will log these errors at a
 // higher log level. We still need to throw these errors to signal that the
 // sync should be retried.
-func IgnorableError(s string, args ...interface{}) ignorableError {
-	return ignorableError(fmt.Sprintf(s, args...))
+func IgnorableError(s string, args ...interface{}) NewIgnorableError {
```

Comment:
This should return a regular `error` instead

Reply:
Thank you for your review and comments, I have modified the ignorableError so it only returns error.